recipes/r-sccore/meta.yaml

```
@@ -0,0 +1,128 @@
+{% set version = '1.0.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sccore
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sccore_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sccore/sccore_{{ version }}.tar.gz
+  sha256: 75220c8309b41598ebc12235e6d16ab8e08a231d86cc2d63e6729c2b99074c6b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppeigen
+    - r-rcppprogress
+    - r-dplyr
+    - r-ggplot2
+    - r-ggrepel
+    - r-igraph
+    - r-irlba
+    - r-magrittr
+    - r-proc
+    - r-pbmcapply
+    - r-rlang
+    - r-scales
+    - r-tibble
+    - r-uwot
+    - r-withr
+
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppeigen
+    - r-rcppprogress
+    - r-dplyr
+    - r-ggplot2
+    - r-ggrepel
+    - r-igraph
+    - r-irlba
+    - r-magrittr
+    - r-proc
+    - r-pbmcapply
+    - r-rlang
+    - r-scales
+    - r-tibble
+    - r-uwot
+    - r-withr
+
+test:
+  commands:
+    - $R -e "library('sccore')"           # [not win]
+    - "\"%R%\" -e \"library('sccore')\""  # [win]
+
+about:
+  home: https://github.com/kharchenkolab/sccore
+  license: GPL-3.0-or-later
```

the cran meta data says `# License: GPL-3`, thus:
```suggestion
  license: GPL-3.0-only
```